requirements/requirements_docs.txt

```diff
@@ -1,4 +1,4 @@
-sphinx~=4.5.0
+sphinx==5.1.0
```

Comment:
@shiftinv why did we change this to be exactly 5.1.0 and not a patch release if one is made? This was intially introduced in b81682824a24ce0ea7a32efbc4964afe6b01c030

Reply:
... not entirely sure, possibly just out of caution to avoid extensions breaking randomly (even if there shouldn't be any breaking changes). I wouldn't mind `~=` either though.

Reply:
Keeping it pinned, then.

Reply:
Just saw the changelog for 5.1.1, actually going to remove the pin 🙃